### PR TITLE
Wanda testing preflight 0.8.2

### PR DESCRIPTION
**Pull request by wanda-carlson**
Wanda testing preflight 0.8.2

### CODE REVIEW

**Comment by wanda-carlson** [comment]
This is the new PR for preflight tests @henry-richard 
Screen Shot 2022-01-06 at 2 57 26 PM

**Comment by wanda-carlson** [comment]
Can you check this **Henry Richard**?

**Comment by wanda-carlson** [comment]
Pullflow is great! @wanda-carlson

**Comment by henry-richard** [comment]
Yes it is @wanda-carlson

File: src/wanda.py [hunk 1]
```diff
@@ -4,3 +4,4 @@
 print("Wanda testing v0.8.0 Preflight 0.1")
 print("Wanda testing notifications 0.2") # old local testing
 print("Wanda testing preflight 0.8.1")
```
Comment (henry-richard) [inline src/wanda.py:6]: Did we ever merge this?

File: src/wanda.py [hunk 2]
```diff
@@ -4,3 +4,4 @@
...
+print("Wanda testing preflight 0.8.2")
```
Comment (henry-richard) [inline src/wanda.py:7]: Please capitalize letters

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by henry-richard — APPROVED** [review]

**Pull request closed without merging**